<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="ChargeIndicator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>192</red>
          <green>97</green>
          <blue>203</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>192</red>
          <green>97</green>
          <blue>203</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>145</red>
          <green>145</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCharging">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plugged In</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>User</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Data Collection</string>
     </attribute>
     <widget class="QCheckBox" name="skinContactChecked">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Skin Contact</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="sliderValue">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>56</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dropdown">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>H1 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H1 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H2 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H2 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H3 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H3 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H4 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H4 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H5 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H5 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H6 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H6 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F1 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F1 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F2 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F2 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F3 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F3 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F4 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F4 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F5 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F5 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F6 Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F6 Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRaDoTech_Simulator">
    <property name="title">
     <string>RaDoTech Simulator</string>
    </property>
   </widget>
   <addaction name="menuRaDoTech_Simulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
